.github/workflows/ci.yml

```diff
@@ -26,36 +27,55 @@ jobs:
     strategy:
       matrix:
         php: [ '7.4', '8.0', '8.1', '8.2' ]
+        browser: [ 'firefox', 'chrome' ]
+        selenium: [ '2.53.1', '3', '4' ]
       fail-fast: false
 
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Start Selenium
+        run: |
+          mkdir -p ./logs
+          SELENIUM_IMAGE=selenium/standalone-${{ matrix.browser }}:${{ matrix.selenium }} docker compose up --no-color &> ./logs/selenium.log &
+
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           coverage: "xdebug"
           php-version: "${{ matrix.php }}"
           ini-file: development
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why this is needed?

Reply:
From what I understand, that disables github rate limiting anonymous Composer calls/downloads.

It's needed in our case due to the large number of parallel jobs. Later we could also enable composer caching though.

Reply:
As mentioned below, I removed it for now.